ui_testing/pages/order_page.py

```diff
@@ -350,8 +350,8 @@ def get_suborder_table(self):
         self.info(' Get suborder table list.')
         self.base_selenium.click(element='order:suborder_list')
 
-    def create_new_suborder(self, material_type='', article_name='', test_plan='', test_unit='',
-                            add_new_suborder_btn='order:add_new_item'):
+    def create_new_suborder(self, material_type='', article_name='', test_plans=[''], test_unit='',
+                            add_new_suborder_btn='order:add_new_item',add_tstunit = True):
```

Comment:
If you updated testplan to be list, you need to find usages of this function and update them .
You can define new function called create suborder with multiple testplans

Reply:
i already sent it a list, if i create new function it will be duplicated i think ?!